src/modules/position_estimator_inav/position_estimator_inav_main.cpp

```diff
@@ -252,14 +277,13 @@ int position_estimator_inav_thread_main(int argc, char *argv[])
 	float eph_mocap = 0.05f;
 	float epv_mocap = 0.05f;
 
-	float x_est_prev[2], y_est_prev[2], z_est_prev[2];
-	memset(x_est_prev, 0, sizeof(x_est_prev));
-	memset(y_est_prev, 0, sizeof(y_est_prev));
-	memset(z_est_prev, 0, sizeof(z_est_prev));
+	Vector2f x_est_prev = {0.0f, 0.0f};
```

Comment:
That's a good question and I'm trying to found the answer without checking the code with/without init to 0. We can also check using [`isZero()`](http://eigen.tuxfamily.org/dox/classEigen_1_1DenseBase.html#a6dc0eb069536157ead1d46c660b9622d).